rules/php80/tests/Rector/Class_/AnnotationToAttributeRector/Fixture/symfony_constraints.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rector\Php80\Tests\Rector\Class_\AnnotationToAttributeRector\Fixture;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class SymfonyConstraints
+{
+    /**
+     * @Assert\Type("string")
+     * @Assert\Length(max="180")
+     * @Assert\Email(groups={"registration"})
+     * @Assert\NotNull(groups={"registration"})
+     */
+    private $email;
+
+    /**
+     * @\Symfony\Component\Validator\Constraints\Type("string")
+     * @\Symfony\Component\Validator\Constraints\Length(max="180")
+     * @\Symfony\Component\Validator\Constraints\Email(groups={"registration"})
+     * @\Symfony\Component\Validator\Constraints\NotNull(groups={"registration"})
+     */
+    private $email2;
```

Comment:
There is too many tested case in single file. Could you separate it per class, so we see which exactly class is covered and which not right from the test case?